tests/api_tests.rs

```diff
@@ -25,6 +33,18 @@ where
     let state = Arc::new(RwLock::new(State {
         expectations: vec![],
     }));
+    let lock = PORT_USED.lock().unwrap();
```

Comment:
Does not this lock that is release only after test is executed will block the concurrent tests ?